### PR TITLE
Ajay tripathy gpu support

### PR DESCRIPTION
**Pull request by AjayTripathy**
Support amazonaws vgpu

### CODE REVIEW

File: pkg/prom/query.go [hunk 1]
```diff
@@ -190,7 +190,7 @@ func (ctx *Context) query(query string) (interface{}, prometheus.Warnings, error
 	statusCode := resp.StatusCode
 	statusText := http.StatusText(statusCode)
 	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
-		return nil, warnings, CommErrorf("%d (%s) URL: '%s' Headers: '%s', Body: '%s' Query: '%s'", statusCode, statusText, req.URL, util.HeaderString(resp.Header), body, query)
+		return nil, warnings, fmt.Errorf("%d (%s) URL: '%s', Request Headers: '%s', Headers: '%s', Body: '%s' Query: '%s'", statusCode, statusText, req.URL, req.Header, util.HeaderString(resp.Header), body, query)
```
Comment (nikovacevic) [inline pkg/prom/query.go:193]: Is this intentional?

Comment (AjayTripathy) [reply pkg/prom/query.go:193]: Bad merge sry. I _do_ want to log request headers here, but I shouldn't change this to fmt.Errorf instead of CommError

**Review by mbolt35 — APPROVED** [review]
LGTM

**Comment by kbrwn** [comment]
test of ticket to commit link

gz#388

**Pull request merged**